web/pages/index.tsx

```diff
@@ -28,7 +27,10 @@ import {
   PopoverBody,
   PopoverArrow,
   PopoverCloseButton,
-  Box,
+  Alert,
+  AlertIcon,
+  AlertTitle,
+  AlertDescription,
```

Comment:
Let's use a toast instead - I'm sorry for suggesting Alert in the ticket description. It'll act more like a popup that disappears over time. Have the toast appear at the top of the screen. I'm so sorry about that!